### PR TITLE
Minor fixes to spellings and output format of Options.ToString()

### PR DESCRIPTION
**Pull request by Wennemyr**

### CODE REVIEW

**Review by sixlettervariables — APPROVED** [review]
Good catch

**Comment by scottf** [comment]
@Wennemyr Is it possible that you sign your commit? https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

File: src/NATS.Client/Options.cs [hunk 1]
```diff
@@ -828,7 +828,7 @@ public override string ToString()
             }
             sb.AppendFormat("SubChannelLength={0};", SubChannelLength);
             sb.AppendFormat("Timeout={0};", Timeout);
-            sb.AppendFormat("Pendantic={0}", Pedantic);
+            sb.AppendFormat("Pedantic={0}", Pedantic);
```
Comment (scottf) [inline src/NATS.Client/Options.cs]: Might as well remove this while we are here, it's just a duplication of line 801

**Review by scottf — CHANGES_REQUESTED** [review]

File: src/NATS.Client/Options.cs (hunk 1 above)
Comment (Wennemyr) [reply src/NATS.Client/Options.cs]: I didn't know about signing, will figure it out before next time. My apologies, and thank you.

**Review by scottf — APPROVED** [review]

**Comment by scottf** [comment]
using admin to merge unsigned commits for extremely minor text change.

**Pull request merged**